python/cuml/ensemble/randomforest.pyx

```diff
@@ -370,7 +370,7 @@ class RandomForestClassifier(Base):
                    number of trees in the forest.
     handle : cuml.Handle
              If it is None, a new one is created just for this class.
-    split_algo : 0 for HIST, 1 for GLOBAL_QUANTILE and 3 for SPLIT_ALGO_END
+    split_algo : 0 for HIST, 1 for GLOBAL_QUANTILE and 2 for SPLIT_ALGO_END
```

Comment:
Spacing looks weird in both old and new versions. Maybe it's just a github problem though?

Reply:
I checked the indentation it is the same for all the variables